kolibri/deployment/default/settings/base.py

```diff
@@ -156,7 +156,7 @@
 # https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
 # http://helpsharepointvision.nevron.com/Culture_Table.html
 
-with open(os.path.join(KOLIBRI_MODULE_PATH, "locale", "supported_languages.json")) as f:
+with open(os.path.join(KOLIBRI_MODULE_PATH, "locale", "supported_languages.json"),encoding="utf8") as f:
```

Comment:
remember to lint your code!

```
pythonlint runtests: commands[0] | flake8 kolibri
kolibri/deployment/default/settings/base.py:159:82: E231 missing whitespace after ','
```

This is easier if you use pre-commit: http://kolibri-dev.readthedocs.io/en/develop/start/getting_started.html#pre-commit-install